node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs');
+
+const X = process.argv[2];
```

Comment:
You don't want to get in the habit of using single character variable names. Here, something like `file`, `filePath`, `fileToRead`, etc. would be better options.